main/logs.js

```diff
@@ -8,20 +8,20 @@ class Logs {
    * Keep last 100 lines of logs for inspection
    * @param {string} line
    */
-  push (line) {
+  pushLine (line) {
     this.#logs.push(line)
     this.#logs.splice(0, this.#logs.length - 100)
   }
 
-  get () {
-    return this.getLast(Infinity)
+  getLines () {
```

Comment:
```suggestion
  get () {
```

or

```suggestion
  getAll () {
```

Since this is returning all, I find the `Lines` part confusing. But I don't feel strongly about this, please ignore if you disagree.

Reply:
We are keeping only the last 100 lines (IIRC). I feel "getAll" would be misleading.

I don't mind to use `get`.